src/participant/participant.ts

```diff
@@ -182,6 +195,33 @@ export default class ParticipantController {
     });
   }
 
+  async askCopilotFromTreeItem(
+    treeItem: DatabaseTreeItem | CollectionTreeItem
+  ): Promise<void> {
+    if (treeItem instanceof DatabaseTreeItem) {
+      const { databaseName } = treeItem;
+
+      await this.sendMessageToParticipant({
+        message: `I want to ask questions about the \`${databaseName}\` database.`,
+        isNewChat: true,
+      });
+    } else if (treeItem instanceof CollectionTreeItem) {
+      const { databaseName, collectionName } = treeItem;
+
+      await this.sendMessageToParticipant({
+        message: `I want to ask questions about the \`${databaseName}\` database's \`${collectionName}\` collection.`,
+        isNewChat: true,
+      });
+    } else {
+      throw new Error('Unsupported tree item type');
+    }
+
+    await this.sendMessageToParticipant({
+      message: '',
+      isPartialQuery: true,
+    });
```

Comment:
What's the purpose of this? As far as I can tell, once we send the first message, it keeps the `@MongodB` scope.